src/wp-includes/blocks.php

```diff
@@ -952,18 +993,12 @@ function insert_hooked_blocks( &$parsed_anchor_block, $relative_position, $hooke
  * @param array                           $parsed_anchor_block The anchor block, in parsed block array format.
  * @param string                          $relative_position   The relative position of the hooked blocks.
  *                                                             Can be one of 'before', 'after', 'first_child', or 'last_child'.
- * @param array                           $hooked_blocks       An array of hooked block types, grouped by anchor block and relative position.
  * @param WP_Block_Template|WP_Post|array $context             The block template, template part, or pattern that the anchor block belongs to.
  * @return string Empty string.
  */
-function set_ignored_hooked_blocks_metadata( &$parsed_anchor_block, $relative_position, $hooked_blocks, $context ) {
```

Comment:
😢 I suspected this might be the case, but hoped it wasn't. As we've discussed privately in DM there is precedent  for deprecating non-last args https://github.com/WordPress/wordpress-develop/blob/df256e414c183ff001cdf26ebf37b19b960180de/src/wp-admin/includes/upgrade.php#L47

Reply:
Updated in:

* https://github.com/WordPress/wordpress-develop/pull/6584/commits/a0807c63a9f46c354b62317c2073c0e21c336435
* https://github.com/WordPress/wordpress-develop/pull/6584/commits/937c667a64f25bcbda05626086c9b7573d477f01
* https://github.com/WordPress/wordpress-develop/pull/6584/commits/dda337cd1248a44461ac0502a2156767aa3c351f
* https://github.com/WordPress/wordpress-develop/pull/6584/commits/98ced4b3d59ba20a059e1a09fde9f142754d4880